### PR TITLE
17 create language model

### PR DESCRIPTION
**Pull request by hassanrad**
Completed, please review the code and let us know if it's okay to merge

### CODE REVIEW

File: spec/features/profile_feature_spec.rb [hunk 1]
```diff
@@ -5,7 +5,9 @@
     scenario  'profile is created automatically' do
       sign_up_user
       click_link 'Your profile'
-      expect(current_path).to eq '/users/1/profiles/1'
+      # expect(current_path).to eq '/users/1/profiles/1'
```
Comment (toby676) [inline spec/features/profile_feature_spec.rb]: ???

File: app/views/profiles/edit.html.erb [hunk 2]
```diff
@@ -1,7 +1,22 @@
 <%= form_for [@user,@profile] do |f| %>
-  <%= f.label :name %>
-  <%= f.text_field :name %>
-  <%= f.label :email %>
-  <%= f.text_field :email %>
-  <%= f.submit %>
+  <div class="field">
```
Comment (Jojograndjojo) [inline app/views/profiles/edit.html.erb]: may you please replace f by form

File: db/migrate/20160720134923_create_languages.rb [hunk 3]
```diff
@@ -0,0 +1,9 @@
+class CreateLanguages < ActiveRecord::Migration[5.0]
+  def change
+    create_table :languages do |t|
+      t.string :language_type
+
```
Comment (Jojograndjojo) [inline db/migrate/20160720134923_create_languages.rb]: may you replace t by language

Comment (Jojograndjojo) [inline db/migrate/20160720134923_create_languages.rb]: and get rid of the space

File: app/views/profiles/edit.html.erb (hunk 2 above)
Comment (chdezmar) [inline app/views/profiles/edit.html.erb]: I think is alright

Comment (toby676) [inline app/views/profiles/edit.html.erb]: nah

**Comment by toby676** [comment]
:shipit:

**Pull request merged**